### DISCUSSION TITLE
OVMF with CSM

### DISCUSSION THREAD

**Discussion by Gong-qg** [cloud-hypervisor/cloud-hypervisor#4935] [Q&A]
Can OVMF with CSM be used for Cloud Hypervisor to start virtual machine images in MBR format？

**Comment by rbradford** [comment]
It's not something we support any longer. But you might be able to get it work. You will need to patch SeaBIOS like this: https://github.com/coreboot/seabios/commit/3fcece5c0eb5135630278cbee23686397a916d36

**Reply by Gong-qg** [reply]
I'll try, thank you!😆
